frontend/src/lib/k8s/api/v2/useKubeObjectList.ts

```diff
@@ -113,43 +113,75 @@ function useWatchKubeObjectLists<K extends KubeObject>({
   lists: Array<{ cluster: string; namespace?: string; resourceVersion: string }>;
 }) {
   const client = useQueryClient();
+  const latestResourceVersions = useRef<Record<string, string>>({});
```

Comment:
I think it makes sense to implement a new `useWatchKubeObjectLists` without these changes so that it can be turned on or off easily later by toggling a constant. `If (ENABLE_WEBSOCKET_MULTIPLEXER)`. What do you think?

This way it's a bit less risky later on. Eventually we can remove it.

Reply:
Maybe @sniok can assist with some tests for the existing useWatchKubeObjectLists?

@knrt10  would that help?

Reply:
@sniok Can you please write some tests for useWatchKubeObjecfLists?

Reply:
sure